### PR TITLE
Nftrwp 136 change search current contacts text

### PR DESCRIPTION
**Pull request by Rajibshrestha59**

### CODE REVIEW

**Review by biswanathsahoo — CHANGES_REQUESTED** [review]
Please resolve the conflict

**Comment by Rajibshrestha59** [comment]
> Please resolve the conflict

Please check now @biswanathsahoo

**Review by biswanathsahoo — CHANGES_REQUESTED** [review]
Validation is not working as expected.
Please check these points 
- On manual contact form either phone number or email is mandatory
- On open the popup add focus on the first name input field
- Add enter key event on each input field which will submit the form
- First name and last name are optional, but if someone is providing ten only we have to validate
-  On closing the popup reset the search box and show all the contact(Now it's resetting but now showing all the contacts)

File: src/common/components/ManualContactPopup.js [hunk 1]
```diff
@@ -115,16 +155,16 @@ const ManualContactPopup=({show,
...
                   type={"name"}
                   InputValue={inputFields.first_name}
                   HandleInputChange={HandleInputChange("first_name")}
-                  onFocus={() => HandleFocus("first_name")}
+                  // onFocus={() => HandleFocus("first_name")}
                   HandelKeyPress={(event)=>{
                     if (event.which === 13 ) {
-                        handleBtnClick()
-                    }}}
+                    validateFirstName(event)}}}
```
Comment (biswanathsahoo) [inline src/common/components/ManualContactPopup.js]: Here enable the enter key event and don't validate the first name on key press

File: src/common/components/ManualContactPopup.js [hunk 2]
```diff
@@ -135,8 +175,7 @@ const ManualContactPopup=({show,
                   onFocus={() => HandleFocus("last_name")}
                   HandelKeyPress={(event)=>{
                     if (event.which === 13 ) {
-                        handleBtnClick()
-                    }}}
+                    validateLastName(event)}}}
```
Comment (biswanathsahoo) [inline src/common/components/ManualContactPopup.js]: Here also, enable the enter key event and don't validate the last name on key press

**Review by biswanathsahoo — CHANGES_REQUESTED** [review]

**Review by biswanathsahoo — APPROVED** [review]

**Pull request merged**